package.json

```diff
@@ -44,8 +46,11 @@
   },
   "size-limit": [
     {
-      "limit": "650 B",
-      "path": "index.js"
+      "name": "core",
+      "import": {
+        "index.js": "{ navigate, changed, key, createRouter }"
```

Comment:
`changed` and `key` import names are too common. Many developers do not know the syntax for import to change the name. It will be better to add some prefix like `routerKey`.

Reply:
I corrected the names for export